doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -4,38 +4,62 @@
 <p align="right"><img src="../figures/bogo_lsf.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [5.1 Introduction](#51-introduction)
-* [5.2 Principles](#52-principles)
-* [5.3 Node Hardening](#53-node-hardening)
-* [5.4 Authentication & Authorisation](#54-authentication--authorisation)
-* [5.5 Use Namespaces to Establish Security Boundaries](#55-use-namespaces-to-establish-security-boundaries)
-* [5.6 Separate Sensitive Workload](#56-separate-sensitive-workloads)
-* [5.7 Create and Define Network Policies](#57-create-and-define-network-policies)
-* [5.8 Run latest Version](#58-run-latest-version)
-* [5.9 Secure Platform Metadata](#59-secure-platform-metadata)
-* [5.10 Enable Logging and Monitoring](#510-enable-logging-and-monitoring)
-* [5.11 Run-time Security](#511-run-time-security)
-* [5.12 Secrets Management](#512-secrets-management)
-* [5.13 Trusted Registry](#513-trusted-register)
-* [5.14 Orchestration & Container Manager](#514-orchestration--container-manager)
-* [5.15 Security Parameters](#515-security-parameters)
-* [5.16 Isolation](#516-isolation)
-    * [5.16.1 VM vs. Container Isolation](5161-vm-vs-container-isolation)
-    * [5.16.2 Container Isolation in Kubernetes Cluster](5162-container-isolation-in-kubernetes-cluster)
-        * [5.16.2.1 Namespaces](51621-namespaces)
+- [5. Security Guidance](#5-security-guidance)
```

Comment:
Please delete as it is already in the chapter heading

Reply:
Done